scripts/docker/Dockerfile

```diff
@@ -79,8 +78,8 @@ RUN make install
 
 # Build Portals 4
 WORKDIR $SOS_INSTALL
-RUN git clone --depth 10 https://github.com/regrant/portals4.git portals4
-WORKDIR portals4
+RUN git clone --depth 10 https://github.com/sandialabs/portals4.git portals4
```

Comment:
This latest commit doesn't work for me - it seems related to the temporary fix in #1203 here:
https://github.com/Sandia-OpenSHMEM/SOS/pull/1203/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR638

Should we make that change to portals4 upstream @markbrown314?

Or we could simply clone portals4 at commit https://github.com/sandialabs/portals4/commit/b9083c85d09c8810b726aef2e3cd19355afb5533

Or simply comment this out?  We're not even using it below...

Reply:
Otherwise, thanks for the patch/update @shivnshshrma !